paddle/fluid/imperative/tracer.h

```diff
@@ -58,6 +58,9 @@ class Tracer {
                const NameVarBaseMap& outs, framework::AttributeMap attrs,
                const platform::Place& place, bool trace_bacward);
 
+  void TraceOp(const std::string& type, const NameVarBaseMap& ins,
```

Comment:
Think about how to remove the first version `TraceOp` in the future.

Reply:
The first version `TraceOp` can be removed when all the legacy layers migrated to new generated op function.